src/Builder/ClassModifier/ClassConstant/ClassConstantModifier.php

```diff
@@ -73,6 +73,14 @@ public function setConstant(
 
         if ($isLiteral) {
             $value = $this->parseSingleValue((string)$value);
+        } elseif (is_array($value)) {
+            foreach ($value as $idx => $val) {
```

Comment:
just `if` would look cleaner.
It is better to extract `foreach` to the extra method with meaningful naming.

Reply:
pareseValue() with early return.